text-notification/index.js

```diff
@@ -0,0 +1,85 @@
+module.exports = class TextNotificationPlugin extends BasePlugin {
+
+    /** Plugin info */
+    static get id()             { return 'zongo.text-notify' }
+    static get name()           { return 'Notify Hosts' }
+    static get description()    { return 'This plugin allows users to notify space hosts via text of their arival in the space' }
+
+    phoneNumberSet = new Set()
```

Comment:
Once you have entered a phone number, then override that field with a different phone number, it will send the notification to both phone numbers

Reply:
Made all the requested changes 👍